### PR TITLE
create aleph alpha integration doc

### PR DESCRIPTION
**Pull request by davidmyriel**
This integration doc is built on top of our basic integration and shows how to do an image search based on a common dataset. I wonder if we should have our own demo, though.

I still need to fix some wording and make it a subsection of aleph alpha.

### CODE REVIEW

**Comment by llogiq** [comment]
I think Aleph Alpha is going to do their own demo, so perhaps we don't need to do the work twice.

**Comment by davidmyriel** [comment]
> Let's keep pages consistent? image

I have already fixed this yesterday.

File: qdrant-landing/content/documentation/integrations/aleph-alpha-search.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+There is no need to curate your datasets and train the models. [Aleph Alpha](https://www.aleph-alpha.com/), already has multimodality and multilinguality already built-in. There is an [official Python client](https://github.com/Aleph-Alpha/aleph-alpha-client) that simplifies the integration. [can be integrated seamlessly](https://qdrant.tech/documentation/integrations/#aleph-alpha).
+
+In order to enable the search capabilities, you need to build the search index to query on. For our purposes, 
+we are going to vectorize the images and store their embeddings along with the filenames, we can then return the most 
+similar files for given query. There are two things we need to set up before we start though:
+
+1. A Qdrant instance has to be running. If you want to launch it locally,
+   [Docker is the fastest way to do that](https://qdrant.tech/documentation/quick_start/#installation).
+2. You need to have a [Aleph Alpha account registered and confirmed](https://app.aleph-alpha.com/). Once it’s done, 
+   the API key has to be created (see: [API Tokens](https://app.aleph-alpha.com/profile)).
```
Comment (llogiq) [inline qdrant-landing/content/documentation/integrations/aleph-alpha-search.md:41]: Perhaps use active voice, i.e. "Once that's done, create an API key (...)" here.

File: qdrant-landing/content/documentation/integrations/aleph-alpha-search.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+Once it’s done we can store the Aleph Alpha API key in a variable and choose the model we’re going to use.
```
Comment (llogiq) [inline qdrant-landing/content/documentation/integrations/aleph-alpha-search.md:43]:
```suggestion
Now we can store the Aleph Alpha API key in a variable and choose the model we’re going to use.
```

**Review by llogiq — COMMENTED** [review]
I've added a few small suggestions. Apart from that, do we go for the "we" form or just let "you", the reader do stuff? I have done both in the past, perhaps @davidmyriel has a clearer vision here.

**Comment by davidmyriel** [comment]
Shortened text in tutorial format and moved to Tutorials. 
Turned to active voice and changed titles.
I decided to keep integrations strictly for basic reference docs. 
More structure for tutorials/integrations covered in a PR later today.

**Pull request merged**